tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -1119,8 +1125,66 @@ def _schedule(self):
+            ]) + num_scheduled_generation_requests
+            responses_list = self.dist.tp_allgather([
+                num_scheduled_context_requests,
+                num_scheduled_generation_requests, num_scheduled_tokens
+            ])
+            all_ranks_num_scheduled_context_requests = [
+                response[0] for response in responses_list
+            ]
+            all_ranks_num_scheduled_generation_requests = [
+                response[1] for response in responses_list
+            ]
+            all_ranks_num_scheduled_tokens = [
```

Ruff (0.12.2)

Local variable `all_ranks_num_scheduled_tokens` is assigned to but never used

Remove assignment to unused variable `all_ranks_num_scheduled_tokens`

(F841)